indexer/altitude_loader.cpp

```diff
@@ -29,8 +29,14 @@ void LoadAndMap(size_t dataSize, ReaderSource<FilesContainerR::TReader> & src, T
 
 namespace feature
 {
-AltitudeLoader::AltitudeLoader(MwmValue const & mwmValue)
+AltitudeLoader::AltitudeLoader(Index const & index, MwmSet::MwmId const & mwmId)
```

Comment:
That would work too. I chose this way for symmetry with FeaturesLoaderGuard: while we know we can reuse handles in the current model (i.e. all routing calls made on one thread), it seems safer to have a separate handle for each involved component exactly for the reason of this PR.